test/spec/simd.wast

```diff
@@ -1050,8 +1050,8 @@
 )
 
 ;; conversions
-(assert_return (invoke "i32x4.trunc_sat_f32x4_s" (v128.const f32x4 42 nan infinity -infinity)) (v128.const i32x4 42 0 2147483647 -2147483648))
-(assert_return (invoke "i32x4.trunc_sat_f32x4_u" (v128.const f32x4 42 nan infinity -infinity)) (v128.const i32x4 42 0 4294967295 0))
+(assert_return (invoke "i32x4.trunc_sat_f32x4_s" (v128.const f32x4 42 nan inf -inf)) (v128.const i32x4 42 0 2147483647 -2147483648))
+(assert_return (invoke "i32x4.trunc_sat_f32x4_u" (v128.const f32x4 42 nan inf -inf)) (v128.const i32x4 42 0 4294967295 0))
```

Comment:
Lot of SIMD changes... did we write this by hand, or did the upstream spec change in the middle, I'm just curious?

Reply:
I think we wrote this one by hand as we were implementing SIMD interpretation. That was probably before there were any upstream spec tests.